src/components/Pursuance/PursuancePage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
```

Comment:
@Spetastian Oh interesting.  Can `Redirect` be used to redirect `/pursuance/:pursuanceId` to `/pursuance/:pursuanceId/tasks` so that the Tasks icon in the menu is green, and so there is just 1 URL at which the task hierarchy can be viewed?

Reply:
Sorry about that. I tried Redirect as a solution but it failed when using route params. I forgot to remove the import.